boards/PDM/Src/Io/Io_Efuse.c

```diff
@@ -0,0 +1,350 @@
+#include <assert.h>
+#include <math.h>
+#include <stdlib.h>
+#include "Io_Efuse.h"
+#include "configs/Io_EfuseConfig.h"
+
+#pragma GCC diagnostic ignored "-Wconversion"
+
+struct Efuse_Context
+{
+    // The SPI handle for the SPI device the E-Fuse is connected to
+    SPI_HandleTypeDef *efuse_spi_handle;
+
+    // The handle to Efuse's chip-select GPIO port
+    GPIO_TypeDef *chip_select_port;
```

Comment:
I think we should change `chip_select_port` to `nss_port` because it tells the user that they are working with an active low slave select. Also less verbose.

```suggestion
    GPIO_TypeDef *nss_port;
```